404.md

```diff
@@ -4,8 +4,10 @@ title: Page not found
 toc: false
 ---
 
+# 404 Not Found
+
 <!-- SPDX-License-Identifier: CC0-1.0 -->
-<!-- SPDX-FileCopyrightText: 2022 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
+<!-- SPDX-FileCopyrightText: 2022-2023 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
```

Comment:
Just out of curiosity, why leave 2022 in there? Why not update everything to 2023?

Reply:
It's mostly a convention and we leave it to show the first year for this file.

Reply:
Ah, that explains why different files have different first years. Thanks for explaining!